src/styles/blocks/card.scss

```diff
@@ -0,0 +1,123 @@
+// #region card
+
+.card {
+  width: 198px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__image {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+  }
+
+  &__text-container {
+    align-self: flex-start;
+    margin: 0 16px;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    padding-top: 40px;
+    color: $main-text-color;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    text-align: left;
+    color: $side-text-color;
+    margin-top: 4px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+
+    &-quantity {
+      font-size: 10px;
+      line-height: 14px;
+      color: $main-text-color;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 24px;
+
+    &-text {
+      font-size: 12px;
+      line-height: 18px;
+      color: $side-text-color;
+    }
+
+    &-value {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $main-text-color;
+    }
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    background-color: $main-color;
+    color: #fff;
+    height: 40px;
+    border-radius: 5px;
+    margin-top: 16px;
+    text-decoration: none;
+
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+
+    &:hover {
+      background-color: transparent;
+      color: $main-color;
+      border: 1px solid $main-color;
```

Comment:
The variable `$main-color` is used here. Ensure it is defined in your SCSS files.